test/image/mocks/funnel_axis.json

```diff
@@ -30,6 +31,7 @@
             ],
             "textposition": "inside",
             "insidetextanchor": "start",
+            "textangle": 0,
```

Comment:
Do we _need_ this line? Isn't `textangle: 0` by default for funnel traces?

Reply:
No we don't need them anymore. Fixed in https://github.com/plotly/plotly.js/pull/3817/commits/a7446899f8c48924509c843a42c9554412a4aa9c.